tools/pre-commit-hooks/check_version_consistency.py

```
@@ -0,0 +1,163 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.15.9)

[warning] The file is executable but no shebang is present

(EXE002)